src/content-tags-drawer/ContentTagsDrawer.jsx

```diff
@@ -26,6 +32,42 @@ const ContentTagsDrawer = () => {
 
   const org = extractOrgFromContentId(contentId);
 
+  const [stagedContentTags, setStagedContentTags] = useState({});
+
+  // Add a content tags to the staged tags for a taxonomy
+  const addStagedContentTag = useCallback((taxonomyId, addedTag) => {
+    setStagedContentTags(prevStagedContentTags => {
+      const updatedStagedContentTags = cloneDeep(prevStagedContentTags);
+      if (!updatedStagedContentTags[taxonomyId]) {
+        updatedStagedContentTags[taxonomyId] = [];
+      }
+      updatedStagedContentTags[taxonomyId].push(addedTag);
```

Comment:
```suggestion
      const updatedStagedContentTags = {
         ...prevStagedContentTags,
         [taxonomyId]: [...(prevStagedContentTags[taxonomyId] ?? []), addedTag],
      };
```

I think this ^ is a simpler, more standard way to write this. It doesn't need `lodash.cloneDeep` and should have higher performance since many of the existing arrays don't need to be cloned.

Reply:
Oh cool, this is much cleaner, I've updated it and the rest to match this standard style.